Bottom: 85da29bafe028daa3c36cba080c34c4669d4af9d
Top:    471aba4b7cfa2c7039a9d7ae6ddc7b5b0280d18f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-03 01:29:52 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index 1e289dc..d5f745a 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -23,12 +23,8 @@
 #ifndef __HIDGL_INCLUDED__
 #define __HIDGL_INCLUDED__
 
-//#define TRIANGLE_ARRAY_SIZE 5461
-//#define TRIANGLE_ARRAY_SIZE 2740
-//#define TRIANGLE_ARRAY_SIZE 8738
 #define TRIANGLE_ARRAY_SIZE 30000
 typedef struct {
-//  GLfloat triangle_array [3 * (3 + 2) * TRIANGLE_ARRAY_SIZE];
   GLfloat *triangle_array;
   unsigned int triangle_count;
   unsigned int coord_comp_count;
